tool/piped-base-okd/Dockerfile

```diff
@@ -1,3 +1,4 @@
+# check=skip=SecretsUsedInArgOrEnv
```

Comment:
The following code was pointed out to me, but I ignored it because it was a false positve.
https://github.com/pipe-cd/pipecd/blob/a5327183a47189ec530cead2d8b7a4093d6895cc/tool/piped-base-okd/Dockerfile#L39

ref. https://docs.docker.com/build/checks/#skip-checks